src/Middlewares/PermissionMiddleware.php

```diff
@@ -3,21 +3,25 @@
 namespace Spatie\Permission\Middlewares;
 
 use Closure;
-use Illuminate\Support\Facades\Auth;
 
 class PermissionMiddleware
 {
     public function handle($request, Closure $next, $permission)
     {
-        if (Auth::guest()) {
+        if (auth()->guest()) {
             abort(403);
         }
 
-        $permission = is_array($permission)
-            ? $permission
-            : explode('|', $permission);
+        if (is_array($permission)) {
```

Comment:
I don't think these changes are necessary. Could you please revert back to the old behaviour?